packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -1,28 +1,123 @@
 import { memoizeFunction } from '../../Utilities';
-import { mergeStyleSets, ITheme } from '../../Styling';
-import { IContextualMenuStyles } from './ContextualMenu.Props';
-
+import { ITheme, mergeStyles } from '../../Styling';
+import { IContextualMenuStyles, IMenuItemStyles } from './ContextualMenu.Props';
+import { getStyles as getContextualMenuStyles, getMenuItemStyles } from './ContextualMenu.styles';
 export interface IContextualMenuClassNames {
+  container?: string;
+  root?: string;
+  list?: string;
+  divider?: string;
```

Comment:
I was thinking we could make these all required instead of all optional. Any reason why some of them need to be optional?

Reply:
+1, if we need to do all-optional-interface then we should use Partial<T>